webapp/src/components/modals/channel_settings/channel_settings_internal.jsx

```diff
@@ -8,7 +8,7 @@ import {Modal} from 'react-bootstrap';
 import ReactSelectSetting from 'components/react_select_setting';
 import FormButton from 'components/form_button';
 import Loading from 'components/loading';
-import {getProjectValues, getIssueValuesForMultipleProjects} from 'jira_issue_metadata';
+import {getProjectValues, getIssueValuesForMultipleProjects} from 'utils/jira_issue_metadata';
```

Comment:
@cpoile I overlooked this in the previous merge, not sure how it passed `make all`

Reply:
oh yah, interesting...